modules/microshift-ingress-controller-config.adoc

```diff
@@ -107,116 +128,158 @@ The `AllowedSubjectPatterns` is an optional value that specifies a list of regul
 |`forwardedHeaderPolicy`
 |Specifies when and how the ingress controller sets the `Forwarded`, `X-Forwarded-For`, `X-Forwarded-Host`, `X-Forwarded-Port`, `X-Forwarded-Proto`, and `X-Forwarded-Proto-Version` HTTP headers. The following values are valid:
 
-* `Append`, preserves any existing headers by specifying that the ingress controller appends them.
-* `Replace`, removes any existing headers by specifying that the ingress controller sets the headers.
+* `Append` is the default value. Preserves any existing headers by specifying that the ingress controller appends them.
+* `Replace` removes any existing headers by specifying that the ingress controller sets the headers.
 * `IfNone` sets the headers set by specifying that the ingress controller sets the headers if they are not already set.
-* `Never`, preserves any existing headers by specifying that the ingress controller never sets the headers.
+* `Never` preserves any existing headers by specifying that the ingress controller never sets the headers.
 
 |`httpCompression`
 |Defines the policy for HTTP traffic compression.
 
-* `httpCompressionmimeTypes` defines a list of MIME types to which compression should be applied. For example, `text/css; charset=utf-8`, `text/html`, `text/*`, `image/svg+xml`, `application/octet-stream`, `X-custom/customsub`, using the format pattern, `type/subtype; [;attribute=value]`. The `types` are: application, image, message, multipart, text, video, or a custom type prefaced by `X-`. To see the full notation for MIME types and subtypes, see link:https://datatracker.ietf.org/doc/html/rfc1341#page-7[RFC1341] (IETF Datatracker documentation).
+|`httpCompression.mimeTypes`
+|Defines a list of MIME types to which compression should be applied.
+
+* For example, `text/css; charset=utf-8`, `text/html`, `text/*`, `image/svg+xml`, `application/octet-stream`, `X-custom/customsub`, using the format pattern, `type/subtype; [;attribute=value]`.
+* Valid `types` are: application, image, message, multipart, text, video, or a custom type prefaced by `X-`. To see the full notation for MIME types and subtypes, see link:https://datatracker.ietf.org/doc/html/rfc1341#page-7[RFC1341] (IETF Datatracker documentation).
 
 |`httpEmptyRequestsPolicy`
-|`httpEmptyRequestsPolicy` describes how HTTP connections are handled if the connection times out before a request is received. Allowed values for this field are `Respond` and `Ignore`. The default value is `Respond`. 
+|Describes how HTTP connections are handled if the connection times out before a request is received. Allowed values for this field are `Respond` and `Ignore`. The default value is `Respond`. Empty requests typically come from load-balancer health probes or preconnects and can often be safely ignored. However, these requests can also be caused by network errors and port scans, so setting this field to `Ignore` can impede detection or diagnosis of network problems and detecting intrusion attempts.
```

Comment:
Per ISG: [Do not use (so) by itself as a conjunction](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#s)

```suggestion
|Describes how HTTP connections are handled if the connection times out before a request is received. Allowed values for this field are `Respond` and `Ignore`. The default value is `Respond`. Empty requests typically come from load-balancer health probes or preconnects and can often be safely ignored. However, these requests can also be caused by network errors and port scans. Ass such setting this field to `Ignore` can impede detection or diagnosis of network problems and detecting intrusion attempts.
```